__tests__/app.test.js

```diff
@@ -198,4 +198,50 @@ describe("GET /api/articles/:article_id/comments", () => {
                 expect(body.msg).toBe("Bad Request")
             })
     })
-})
\ No newline at end of file
+})
+
+describe("POST /api/articles/:article_id/comments", () => {
+    test("returns object containing the posted comment", () => {
+        return request(app)
+            .post("/api/articles/2/comments")
+            .send({ username: "lurker", body: "test_body"})
+            .expect(201)
+            .then(({ body }) => {
+                expect(body.comment).toMatchObject({
+                    comment_id: expect.any(Number),
+                    author: "lurker",
+                    body: "test_body",
+                    article_id: 2,
+                    votes: 0,
+                    created_at: expect.any(String)
+                })
+            })
+    })
+    test("returns 400 bad request error when username does not exist", () => {
```

Comment:
great job on these tests there are a few we can add: 
201 - ignores unnecessary  properties 
400 - not an id